loadtest/loadtest_client.go

```diff
@@ -265,7 +265,7 @@ func (c *LoadTestClient) SendTxs(
 				fmt.Printf("Failed to acquire semaphore: %v", err)
 				break
 			}
-			if tx.TxBytes != nil && len(tx.TxBytes) > 0 {
+			if len(tx.TxBytes) > 0 {
```

Comment:
is this intentional? seems unrelated

Reply:
Ah yup this was causing a lint issue that was blocking the PR. Tony makes the same change in his PR here: https://github.com/sei-protocol/sei-chain/pull/1818/files